### PR TITLE
[WNC-35] add get transactions for customer & refractor swagger for sa…

### PR DESCRIPTION
**Pull request by Tricker14**
…ved-receiver

### CODE REVIEW

File: internal/service/implement/transaction_service.go [hunk 1]
```diff
@@ -238,3 +238,25 @@ func (service *TransactionService) InternalTransfer(ctx *gin.Context, transferRe
 	// notify, response history
 	return existsTransaction, nil
 }
+
+func (service *TransactionService) GetTransactions(ctx *gin.Context) ([]entity.Transaction, error) {
+	customerId, exists := ctx.Get("userId")
```
Comment (VuKhoa23) [inline internal/service/implement/transaction_service.go:243]: please use middleware.GetUserIdHelper

File: internal/service/implement/transaction_service.go [hunk 2]
```diff
@@ -238,3 +238,25 @@ func (service *TransactionService) InternalTransfer(ctx *gin.Context, transferRe
...
+	if !exists {
+		return nil, errors.New("customer not exists")
+	}
```
Comment (VuKhoa23) [inline internal/service/implement/transaction_service.go]: please use middleware.GetUserIdHelper

Comment (Tricker14) [reply internal/service/implement/transaction_service.go]: done

**Review by hoadang0305 — APPROVED** [review]
ok

File: internal/controller/http/v1/customer_handler.go [hunk 3]
```diff
@@ -76,3 +79,28 @@ func (h *CustomerHandler) GetNotifications(c *gin.Context) {
...
+func (h *CustomerHandler) GetTransactions(c *gin.Context) {
+	customerId := middleware.GetUserIdHelper(c)
+
+	transactions, err := h.transactionService.GetTransactions(c, customerId)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, httpcommon.NewErrorResponse(
+			httpcommon.Error{
+				Message: err.Error(),
+			},
+		))
```
Comment (VuKhoa23) [inline internal/controller/http/v1/customer_handler.go:100]: messing return

**Pull request merged**